unit_tests/fields/multi_field_test.cpp

```diff
@@ -55,3 +59,68 @@ TEST(TransposeTo, field_constant) {
 		EXPECT_DOUBLE_EQ( rec_t.val<double>("value"), rec_c.val<double>("value") );
 	}
 }
+
+
+
+string eq_data_input = R"JSON(
+[
+  { id=37,
+    a=1,
+    b=0
+  }
+] 
+}
+)JSON";
+
+TEST(Operators, assignment) {
+    Profiler::initialize();
+
+    FilePath::set_io_dirs(".",UNIT_TESTS_SRC_DIR,"",".");
+    FilePath mesh_file("mesh/simplest_cube.msh", FilePath::input_file);
+    GmshMeshReader msh_reader(mesh_file);
+    Mesh * mesh = new Mesh;
+    msh_reader.read_physical_names(mesh);
+	msh_reader.read_mesh(mesh);
+	mesh->check_and_finish();
+
+	std::vector<string> component_names = { "comp_0", "comp_1", "comp_2" };
+
+	MultiField<3, FieldValue<3>::Scalar> mf_base;
+	mf_base.name("a");
+	mf_base.set_components(component_names);
+	mf_base.units( UnitSI::dimensionless() );
+
+	FieldSet set_of_field;
+	set_of_field += mf_base;
+    Input::Type::Array list_type = Input::Type::Array(set_of_field.make_field_descriptor_type("MultiFieldTest"));
+    Input::ReaderToStorage reader( eq_data_input, list_type, Input::FileFormat::format_JSON);
+    Input::Array in_list=reader.get_root_interface<Input::Array>();
+    set_of_field.set_input_list(in_list);
+
+    MultiField<3, FieldValue<3>::Scalar> mf_assignment;
+	EXPECT_EQ("", mf_assignment.name());
+	EXPECT_FALSE(mf_assignment.is_bc());
+
+	// copies
+	mf_assignment
```

Comment:
mf_assignment.set_components
mf_assignment = mf_base
mf_assignment.setup_components
EXPECT_EQ ... for component names
mf_assignment =mf_base
EXPECT_EQ ... for component names